src/util/generic-tests.jsx

```diff
@@ -121,7 +121,7 @@ export function common(Component, {
 					if (shallowExample.is(Component.displayName)) {
 						expect(shallowToJson(shallow(<Component {...shallowExample.props()} />))).toMatchSnapshot();
 					} else {
-						shallowExample.find(Component).forEach((example) => {
+						shallowExample.find(Component.displayName).forEach((example) => {
```

Comment:
It's only an issue for hybridComponents, because the instance that gets passed in to `common` tests generally is the Dumb version rather than the version from `index`